src/autoload/Pyegi.lua

```diff
@@ -33,19 +34,28 @@ local default_settings = {
 local function macro_init() -- aegisub is nil on script's load
 	ADP = aegisub.decode_path
 	ADD = aegisub.dialog.display
+	APT = aegisub.progress.task
 	config_dir = ADP("?user").."/automation/config/Pyegi/"
 	scripts_dir = config_dir.."PythonScripts/"
 	settings_filepath = config_dir.."settings.json"
+	filter_term = ""
 end
 
 -- Source: https://stackoverflow.com/a/11130774
-local function scandir(directory)
+local function scandir(directory, filter_t)
```

Comment:
Since scandir func is used for looking for folders and scandir_f func is used for looking for files and since we might end up using more generalized names for python script names than just "main.py" this scandir_f func may still be needed.

Reply:
hmm. if that's the case, I think `filename` inside scanddir should be `dirname`